test/test_helper.rb

```diff
@@ -6,6 +6,7 @@
 require 'minitest/spec'
 require 'minitest/mock'
 require "minitest/reporters"
+require 'mocha/minitest'
```

Comment:
Why mocha? We now have 2 ways to stub/mock.

Reply:
Because minitest stubbing is insufficient to handle our use case.

Reply:
I'd prefer to get 4d26925692cda13fb52f0382686d0be72be6655c in to avoid the use of mocha.